doc/build-openbsd.md

```diff
@@ -1,158 +1,124 @@
 # OpenBSD build guide
-(updated for OpenBSD 5.7)
+(Tested with OpenBSD 6.6)
 
 This guide describes how to build bitcoind and command-line utilities on OpenBSD.
 
 As OpenBSD is most common as a server OS, we will not bother with the GUI.
 
+**Important**: use `gmake`, not `make`. The non-GNU `make` will exit with a horrible error.
+
 ## Preparation
 
 Run the following as root to install the base dependencies for building:
 
 ```bash
-pkg_add gmake libtool libevent
+pkg_add gmake libtool libevent git boost
 pkg_add autoconf # (select highest version, e.g. 2.69)
-pkg_add automake # (select highest version, e.g. 1.15)
-pkg_add python # (select version 2.7.x, not 3.x)
-ln -sf /usr/local/bin/python2.7 /usr/local/bin/python2
+pkg_add automake # (select highest version, e.g. 1.16)
```

Comment:
here we need to select `automake` 1.16.1, which is not the same as highest version which is `1.9.6p12`.

Selecting the highest version will give you this error:

```
 obsd-bu# ./autogen.sh                                                                                                                                                    
/usr/local/bin/aclocal[35]: /usr/local/bin/aclocal-1.16: not found                                                                                                                                                                                                                
autoreconf-2.69: aclocal failed with exit status: 127
```

Reply:
Isn't this related more to the way `pkg_add` displays the version numbers?

1.9.6 = 1.09.6 < 1.16.1

I will be happy to change the wording though. Would 
`pkg_add automake # (select version 1.16)`
work for you?

Reply:
I see, brain fart of mine, sorry for the noise then. Leave like it is, I was just assuming `pkg_add` sort the version properly whic is not the case.